Documentation/data-collection.md

```diff
@@ -513,6 +513,38 @@ data:
     # consumers: (@openshift/openshift-team-cluster-manager)
     - '{__name__="rhmi_status"}'
     #
+    # owners: (https://github.com/integr8ly, @boomatang)
+    #
+    # rhoam_state captures the currently installed/upgrading RHOAM versions.
+    # Possible label values are:
+    #   status= in progress|complete
+    #   upgrading= true|false
+    #   version= x.y.z
+    # This metric is used by cs-SRE to gain insights into RHOAM version.
+    #
+    # consumers: (@openshift/openshift-team-cluster-manager)
+    - '{__name__="rhoam_state"}'
```

Comment:
it would be nice to have a recording rule keeping only the useful labels, e.g. `max by(status, upgrading, version) (rhoam_state)`